### DISCUSSION TITLE
If you ever decide to create anything regarding Gentoo, then I'd subscribe monthly instantly.

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#486] [General]
Hey Gecko.

Tinkering around with other distros like Gentoo has made me really, really appreciate the hard work you've put into Spiral Linux. I'm currently in the post-install phase of Gentoo right now and trying to set it up how I like on Spiral Linux.

I feel like I'm following behind you while I'm trying to setup my own Gentoo installation. Literally trying to mirror/copy Spiral's setup for Gentoo. So now I have to do everything myself. And getting Snapper setup cleanly and correctly? That s#^&'s hard. Tedious, time-consuming, and difficult.

I wish the Linux community at large had a better appreciation for a project like this. You make a quality software product (regardless if you're using someone else's distro like Debian, that never mattered to me).

I don't think you'd ever be interested in a Gentoo install guide (or making a "Spiral Gentoo" ISO) as that would probably be very time consuming and very draining considering you're already supporting Gecko Linux and Spiral Linux in your free time.

But damn, dude. If you ever changed your mind, you'd have me as a GitHub Sponsor instantly. I depend on Spiral Linux daily.

This was more of an appreciation post than an invitation to Gentoo. But I just wanted to say that I've learned just how much hard work it requires to create your own distro and support it.

Thanks, Geck.

**Comment by geckolinux** [comment]
Hey there, really appreciate your kind words of gratitude, you really made my day!
